### PR TITLE
Add Winter Overhaul

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
As well as related cleaning information.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -5798,3 +5798,12 @@ plugins:
...
       - 'Farm_DLCworkshop03.esp'
+
+  - name: 'Winter Overhaul.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/28665' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0xFDDC0A09
+        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 110
+        udr: 63
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
        udr: 63

```

**Review by MacSplody — CHANGES_REQUESTED** [review]
Needs a newline at the end of file.

File: masterlist.yaml (hunk 1 above)
Comment (Superfly-Johnson) [reply masterlist.yaml]: Done. Should be resolved.

Swift edit, same problem with the github file viewer. Do tell me if the newline didn't actually push,.

**Comment by MacSplody** [comment]
@Superfly-Johnson There's a merge conflict with v0.15, so the pr branch needs to be rebased onto v0.15 and manually fixed.

**Comment by Superfly-Johnson** [comment]
Merge conflicts should be resolved.

**Review by MacSplody — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -5833,6 +5833,17 @@ plugins:
       - 'Farm_DLCNukaWorld.esp'
       - 'Farm_DLCworkshop03.esp'
 
+  - name: 'Winter Overhaul.esp'
```
Comment (MacSplody) [inline masterlist.yaml:5836]: May as well add the location for future maintenance.

```suggestion
  - name: 'Winter Overhaul.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/28665/' ]
```

Comment (Superfly-Johnson) [reply masterlist.yaml:5836]: Should be solved. The original PR had it, dunno what I did there

**Comment by MacSplody** [comment]
@Superfly-Johnson Thanks.

**Pull request merged**